src/bootstrap/src/core/builder.rs

```diff
@@ -1963,6 +1963,12 @@ impl<'a> Builder<'a> {
             rustflags.arg("-Ccontrol-flow-guard");
         }
 
+        // Same for EHCont Guard (this is not combined with the previous if-statement to make
+        // merges with upstream easier).
```

Comment:
nit: Comment should probably be adjusted 🙂

Reply:
Updated comment.  Thanks!